astroquery/alma/tests/test_alma.py

```diff
@@ -97,7 +98,7 @@ def test_SgrAstar(monkeypatch):
     result = alma.query_object('eta carinae')
 
     assert len(result) == 15
-    assert b'2011.0.00497.S' in result['Project code']
```

Comment:
Huh... No need to check for astropy version here?

Reply:
nope, somehow this seems to work as is. There are ongoing changes on the alma side, so I suspect this might have been because of that